models/user.js

```diff
@@ -0,0 +1,37 @@
+const mongoose = require('mongoose');
+
+//user schema 
+const userSchema=new mongoose.Schema({
+    username:
+    {
+        type: String,
+        unique:true
+    },
+
+    password:String,
+
+    email:
+    {
+    type: String,
+    unique:true
+    },
+
+    firstname: String,
+    lastname: String,
+
+    addresses: [
```

Comment:
please use address as a diffrent model and create relationship between them, users and address column